iit/model_pairs/base_model_pair.py

```diff
@@ -18,6 +20,24 @@
 from iit.utils.index import Ix, TorchIndex
 from iit.utils.metric import MetricStoreCollection, MetricType
 
+def in_notebook() -> bool:
```

Comment:
I think we can do this by just importing tqdm: much cleaner that way. (at least according to [this](https://stackoverflow.com/questions/42212810/tqdm-in-jupyter-notebook-prints-new-progress-bars-repeatedly))